src/qt/bitcoin.cpp

```diff
@@ -484,16 +486,25 @@ void BitcoinApplication::initializeResult(bool success)
         window->setClientModel(clientModel);
 
 #ifdef ENABLE_WALLET
-        // TODO: Expose secondary wallets
-        if (!vpwallets.empty())
-        {
-            walletModel = new WalletModel(platformStyle, vpwallets[0], optionsModel);
+        bool fFirstWallet = true;
+        for (CWalletRef pwallet : vpwallets) {
+            WalletModel * const walletModel = new WalletModel(platformStyle, pwallet, optionsModel);
 
-            window->addWallet(BitcoinGUI::DEFAULT_WALLET, walletModel);
-            window->setCurrentWallet(BitcoinGUI::DEFAULT_WALLET);
+            QString WalletName = QString::fromStdString(pwallet->GetName());
+            if (WalletName.endsWith(".dat")) {
```

Comment:
Hmm.. not sure if we should hardcode `.dat` at this point.

Reply:
It's strictly for stripping.

Reply:
https://github.com/bitcoin/bitcoin/pull/11383#discussion_r140410333

> Hmm.. not sure if we should hardcode .dat at this point.

Note: this is now cleaned up in a later commit ("Get wallet name from WalletModel rather than passing it around")